Update plugins.md (#4304)
webpack/webpack.js.org: src/content/concepts/plugins.md

};
```

> The ProgressPlugin created above can be used to customize how progress is reported during compilation. The HtmlWebpackPlugin will generate a file (specified in template) that includes all webpack bundles generated.

Reviewer: ```suggestion
The `ProgressPlugin` is used to customize how progress should be reported during compilation, and `HtmlWebpackPlugin` will generate a HTML file including the `my-first-webpack.bundle.js` file with a `script` tag.
```
